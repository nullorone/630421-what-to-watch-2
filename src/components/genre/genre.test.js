import React from "react";
import renderer from "react-test-renderer";
import Genre from "./genre";

describe(`Make snapshot`, () => {
  const initProps = {
    genre: `Academy`,
    isSelected: false,
    onGenreClick: jest.fn(),
  };

  it(`Get snapshot component`, () => {
    const tree = renderer
      .create(
          <Genre
            {...initProps}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
