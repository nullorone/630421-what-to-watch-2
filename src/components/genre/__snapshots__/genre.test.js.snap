// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make snapshot Get snapshot component 1`] = `
<li
  className="catalog__genres-item false"
>
  <a
    className="catalog__genres-link"
    href="#"
    onClick={[Function]}
  >
    Academy
  </a>
</li>
`;
